hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/security/ssl/PemFileBasedKeyStoresFactory.java

```diff
@@ -51,7 +51,7 @@ public class PemFileBasedKeyStoresFactory implements KeyStoresFactory,
   /**
    * Default format of the keystore files.
    */
-  public static final String DEFAULT_KEYSTORE_TYPE = "jks";
+  public static final String DEFAULT_KEYSTORE_TYPE = KeyStore.getDefaultType();
```

Comment:
As I see we use the constant only internally in this class, can we either inline this, or make it private? I prefer inlining the call, even though it means calculating the value every time, but the trust/keystores are cached so it does not happen often, but help us to see all the usages right away when we check for this method within the code

Reply:
I've inlined the usages. I agree with it being easier to search that way, which might or might not come in handy later.